### PR TITLE
Fix condition in HelpFunc

### PR DESCRIPTION
**Pull request by bogem**
I think, it's more obvious now, what does this method

### CODE REVIEW

**Comment by eparis** [comment]
I think you broke it. but I might not understand. You are only looking at parent. The previous code checked any ancestor.

**Comment by bogem** [comment]
@eparis you're right. I fixed it

File: command.go [hunk 1]
```diff
@@ -247,6 +243,19 @@ func (c *Command) HelpFunc() func(*Command, []string) {
 	}
 }
 
+// checkHelpFunc checks if there is helpFunc in ancestors of c.
+func (c *Command) checkHelpFunc() func(*Command, []string) {
+	if c != nil {
```
Comment (eparis) [inline command.go]: Preference is for
```
if c == nil {
  return nil
}
```

File: command.go [hunk 2]
```diff
@@ -247,6 +243,19 @@ func (c *Command) HelpFunc() func(*Command, []string) {
...
+		if c.helpFunc != nil {
+			return c.helpFunc
+		}
+		if c.parent != nil {
```
Comment (eparis) [inline command.go]: I believe this should be `if c.HasParent()`

**Pull request merged**